.devcontainer/devcontainer.json

```diff
@@ -12,8 +11,11 @@
     "vscode": {
       // Set *default* container specific settings.json values on container create.
       "settings": {
-        "python.defaultInterpreterPath": "/usr/local/bin/python",
-        "python.formatting.blackPath": "/usr/local/py-utils/bin/black"
```

Comment:
Will Black be inferred now based on the .venv interpreter path?

Reply:
This is a deprecated setting that I'm removing. The vscode black integration is now handled using a separate extension instead of the python extension. The main way our dev process interacts with black is using the pre-commit hooks which aren't impacted by this.